docs/src/main/asciidoc/security-oidc-code-flow-authentication-concept.adoc

```diff
@@ -217,7 +608,7 @@ If the `quarkus.oidc.logout.post-logout-path` is set then a `q_post_logout` cook
 
 Note that a cookie name will vary when using xref:security-openid-connect-multitenancy.adoc[OpenID Connect Multi-Tenancy]. For example, it will be named `q_post_logout_tenant_1` for a tenant with a `tenant_1` ID, and so on.
```

Comment:
```suggestion
Note that a cookie name varies when using xref:security-openid-connect-multitenancy.adoc[OpenID Connect Multi-Tenancy]. For example, it will be named `q_post_logout_tenant_1` for a tenant with a `tenant_1` ID, and so on.
```